src/jarabe/main.py

```diff
@@ -58,7 +58,6 @@
 gi.require_version('Wnck', '3.0')
 gi.require_version('SugarExt', '1.0')
 gi.require_version('GdkX11', '3.0')
-gi.require_version('WebKit', '3.0')
```

Comment:
No, my patch uses WebKit 3.0 (also known as WebKit1), and WebKit2 4.0, it does not use WebKit 4.0; there's no such typelib.

A try-except is a conditional import.  My patch does have a conditional import, see [viewhelp.py#L43](https://github.com/quozl/sugar/blob/0c4d099bd8c612978ce4f9de4e1d10cf2e4227f5/src/jarabe/view/viewhelp.py#L43).

The code you've highlighted with your comment was for WebKit1; it should have been removed already, but Sam didn't remove it when he took WebKit1 out of the code base and broke Sugar on Fedora 18 and Fedora 20 (552cf3ff0489ab867282230b14fe3ca38da82d43).

What happens when you try sugar-build?  That's the key.  There is an out of date reference to WebKit1 3.0 in sugar-build; [root.json](https://github.com/sugarlabs/sugar-build/blob/master/build/root.json#L83) contains `webkitgtk3` dependency under Fedora 23 (!).

So sugar-build is out of date with sugar; it needs maintenance.  Please update sugar-build to depend on WebKit2 4.0 (to match sugar HEAD), and also a distribution after Fedora 23?  May I suggest Fedora 25 or Rawhide?  I'd do it myself but I don't run sugar-build; it's evil.  :grin:

References:
* [Fedora 23 webkitgtk3 source yields the WebKit 3.0 typelib](https://src.fedoraproject.org/cgit/rpms/webkitgtk3.git/tree/webkitgtk3.spec?h=f23#n175) (this is WebKit1 with Gtk+3)
* [Fedora 25 webkitgtk4 source yields the WebKit2 4.0 typelib](https://src.fedoraproject.org/cgit/rpms/webkitgtk4.git/tree/webkitgtk4.spec?h=f25#n216) (this is WebKit2 with Gtk+3)